src/api.ts

```diff
@@ -46,13 +46,13 @@ class ChatGPT {
    * @param message - The plaintext message to send.
    * @param opts.conversationId - Optional ID of the previous message in a conversation
    */
-  async sendMessage(conversation: Conversation): Promise<Required<Conversation>> {
+  async sendMessage(conversation: Conversation, action: string): Promise<Required<Conversation>> {
```

Comment:
```suggestion
  async sendMessage(conversation: Conversation, action: 'next' | 'variant' | 'continue'): Promise<Required<Conversation>> {
```

Would be better for the typing

Reply:
And you can consider to put action into `Conversation` type as well.

Reply:
I don't think it's necessary. Because I think `conversation` represents the context of the chat, and I don't think `action` is part of the chat context, so I put it in the parameters of `sendMessage`.